### PR TITLE
Update Makefile to work for generic LDAP groups

### PR DESCRIPTION
**Pull request by accorvin**
This is the first in a series of pull requests that will update our group membership properties file to pull from rover groups.

### CODE REVIEW

File: kfdefs/base/trino/trino-group-mapping.properties [hunk 1]
```diff
@@ -1,4 +1,5 @@
 data-hub-admins:aasthana,acorvin,gfrasca,jvaikath,lferrnan,mshah,rmartine,trino-admin
+data-hub-openshift-admins:aasthana,acorvin,ddalvi,gfrasca,hukhan,hveeradh,jvaikath,lferrnan,rmartine,shgriffi
```
Comment (anishasthana) [inline kfdefs/base/trino/trino-group-mapping.properties:2]: Should Humair, Hema, Lucas, Sherard, and Joseph be in this list?

Comment (accorvin) [reply kfdefs/base/trino/trino-group-mapping.properties:2]: Probably not. This is the current state of the group in Rover though - I plan to remove them from the group and then run the makefile again which will remove them.

**Review by gmfrasca — COMMENTED** [review]
/lgtm

**Review by anishasthana — APPROVED** [review]

**Pull request merged**